docker-compose.yml

```diff
@@ -38,11 +38,12 @@ services:
         max-file: "10"
 
   cardano-db-sync:
-    image: inputoutput/cardano-db-sync:10.0.0
+    image: inputoutput/cardano-db-sync:10.0.1
```

Comment:
Changed to: 
```
image: inputoutput/cardano-db-sync:384a477189d3bb9b16b8960b3bc5c9c91b529220
```

Reply:
What release is this for?

Reply:
If we want this for the `10.0.2` release then we should not be making this change on `master`.

Reply:
Should I just revert the version change? This needs to be targeted at `master` given the [original change](https://github.com/input-output-hk/cardano-db-sync/pull/712) was merged there

Reply:
Done

Reply:
The recommendation is that nobody except the devs and the brave should be running `master`.

Once this PR is merged to `master` (without the version change) the relevant commits will be cherry picked to the `release/10.0.x` branch and then that version number will be bumped to `10.0.2`.